### PR TITLE
add new apis for addon config attribute

### PR DESCRIPTION
**Pull request by ashishmax31**
API changes required to implement: https://issues.redhat.com/browse/SDA-5150

Signed-off-by: ashish <asnaraya@redhat.com>

### CODE REVIEW

File: model/clusters_mgmt/v1/add_on_version_type.model [hunk 1]
```diff
@@ -35,6 +35,9 @@ class AddOnVersion {
 	// List of sub operators for this add-on version.
 	SubOperators []AddOnSubOperator
 
+	// Config for the addon
```
Comment (ciaranRoche) [inline model/clusters_mgmt/v1/add_on_version_type.model:38]: Can you expand on this comment, a little more, it makes sense to me as I have context of what this `config` is but for someone with no context this is a bit abstract.

File: model/clusters_mgmt/v1/env_object_type.model [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
```
Comment (ciaranRoche) [inline model/clusters_mgmt/v1/env_object_type.model:2]: 2022 :smile:

File: model/clusters_mgmt/v1/add_on_config_type.model [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
```
Comment (ciaranRoche) [inline model/clusters_mgmt/v1/add_on_config_type.model:2]: just bump the date to 2022

**Comment by ashishmax31** [comment]
@ciaranRoche addressed the review comments, please have a look now.

**Pull request merged**